h2o-docs/src/product/blog-releases.rst

```diff
@@ -1,325 +1,578 @@
-                                                  treatment_column=treatment_column,
-                                                  uplift_metric="KL",
-                                                  min_rows=10,
-                                                  seed=1234,
-                                                  auuc_type="qini")
-    uplift_model.train(x=predictors, 
-                       y=response, 
-                       training_frame=train, 
-                       validation_frame=valid)
-
-    # Eval performance:
-    perf = uplift_model.model_performance()
-
-    # Generate predictions on a validation set (if necessary)
-    pred = uplift_model.predict(valid)
-
-    # Plot Uplift curve from performance
-    perf.plot_uplift(metric="gain", plot=True)
-
-.. figure:: /images/blog/uplift-blog.png
-    :alt: Uplift Graph
-	:scale: 40%
-    :align: center
-
-Infogram & Admissible Machine Learning
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The `Infogram <admissible.html>`__ reduces bias in built models by taking protected variables (e.g., race and gender) into account. It does this by measuring the admissibility of a variable. This is determined by a safety and relevancy index which serves as a diagnostic tool for fairness. When variables are determined to be unsafe, they will be considered inadmissible.
+    .. code-tab:: r R
+
+        library(h2o)
+        h2o.init()
+
+        # Import the prostate dataset:
+        prostate <- h2o.importFile("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate.csv")
+
+        # Set the target variable:
+        target_variable <- 'CAPSULE'
+        prostate[target_variable] <- as.factor(prostate['CAPSULE'])
+
+        # Split the dataset into train and test:
+        splits <- h2o.splitFrame(data = prostate, ratios = 0.7, seed =1)
+        train <- splits[[1]]
+        test <- splits[[2]]
+
+        # Build and train the model:
+        h2o_dt <- h2o.decision_tree(y = target_variable, training_frame = train, max_depth = 5)
+
+        # Predict on the test data:
+        h2o_pred <- h2o.predict(h2o_dt, test)$predict
+
+    .. code-tab:: python
+
+        import h2o
+        from h2o.estimators import H2ODecisionTreeEstimator
+        h2o.init()
+
+        # Import the prostatedataset:
+        prostate = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate.csv")
+
+        # Set the target variable:
+        target_variable = 'CAPSULE'
+        prostate[target_variable] = prostate[target_variable].asfactor()
+
+        # Split the dataset into train and test:
+        train, test = prostate.split(ratios=[.7])
+
+        # Build and train the model:
+        sdt_h2o = H2ODecisionTreeEstimator(model_id="decision_tree.hex", max_depth=5)
+        sdt_h2o.train(y=target_variable, training_frame=train)
+
+        # Predict on the test data:
+        pred_test = sdt_h2o.predict(test)
+
+GLM AIC and Log Likelihood Implementation
```

Comment:
change

GLM AIC and Log Likelihood Implementation

to 

GLM AIC and Log Likelihood Implementation (Yuliia Syzon)